packages/webdriver/webdriver.d.ts

```diff
@@ -2504,21 +2504,21 @@ declare namespace WebDriver {
          * Receive request information about requests that match the mocked resource.
          * https://wiki.saucelabs.com/display/DOCS/Custom+Sauce+Labs+WebDriver+Extensions+for+Network+and+Log+Commands
          */
-        getMockCalls(mockId: string): ProtocolCommandResponse;
+        getMockCalls(mockId: number): ProtocolCommandResponse;
```

Comment:
Tests require `number` so should this be changed?

Reply:
it should be a string if these commands have been defined as strings before

Reply:
Then I guess all tests here have to be fixed to assert agains string, right?
https://github.com/webdriverio/webdriverio/blob/master/packages/webdriverio/tests/utils/interception/webdriver.test.js#L32

Reply:
Apparently not, I might have made some confusion while I was developing, changed to string now and everything worked